src/docs/config/overview.md

```diff
@@ -160,12 +161,22 @@ Please see the [Dev-Server docs](/docs/dev-server).
 
 *default: `undefined`*
 
-The `preamble` configuration is a `string` that represents a preamble in the main file of the build. Help to persist a banner or add relevant information about the resulting build.
+Used to help to persist a banner or add relevant information about the resulting build, the `preamble` configuration 
+field is a `string` that will be converted into a pinned comment and placed at the top of all emitted JavaScript files,
```

Comment:
Something I would possibly note here is that the preamble doesn't get placed on resultant framework binding files as of today.

Reply:
Not sure how best to word this today though... Possibly as a note underneath the example? Or maybe it's not worth calling out quite yet. I don't mind - your call.

Reply:
I'd lean omitting it for now, but a great point!